pkg/cmd/infra/deployer/deployer.go

```diff
@@ -78,7 +78,7 @@ func NewCommandDeployer(name string) *cobra.Command {
 }
 
 // NewDeployer makes a new Deployer from a kube client.
-func NewDeployer(client kclient.Interface) *Deployer {
+func NewDeployer(client *kclient.Client) *Deployer {
```

Comment:
To be clear, if there was some interface to pod log streams, I'd certainly use it...